spinn_machine/virtual_machine.py

```diff
@@ -163,7 +169,7 @@ def __init__(
         # Assign "IP addresses" to the Ethernet chips
         for i, (x, y) in enumerate(ethernet_chips):
             (a, b) = divmod(i + 1, 128)
-            new_chip = self._create_chip(x, y, "127.0.{}.{}".format(a, b))
+            new_chip = self._create_chip(x, y, "127.0.{}.{}".format(a, b), )
```

Comment:
Is this meant to be here?

Reply:
removed. was thinking need to move stuff, till i found they were self based